app/controllers/ansible_repository_controller.rb

```diff
@@ -49,6 +48,8 @@ def button
       end
     when "ansible_repository_tag"
       tag(self.class.model)
+    when "embedded_configuration_script_payload_tag" # playbooks from nested list
+      tag(model_string_to_constant(@display))
```

Comment:
can you pass a constant here? (rather than trusting whatever might be in `@display`)